src/components/plots/barPlot.ts

```diff
@@ -519,7 +519,7 @@ export module Plots {
       return barPixelWidth;
     }
 
-    public entities(datasets = this.datasets()): Plots.Entity[] {
+    public entities(datasets = this.datasets()): Plots.PlotEntity[] {
```

Comment:
Some of this confusion is because on `Plot` we have to say `Plots.PlotEntity` (since `Plot` is outside of the `Plots` module), but not on any of the subclasses.